static/templates/template/components/iframe-edit.html

```diff
@@ -0,0 +1,30 @@
+<ng-include src="'/static/templates/template/components/question-edit.html'"></ng-include>
+<div layout="row" layout-align="start center">
+    <iframe class="remote-content" ng-src="{{instance.getTrustedUrl(item.aux_attributes.src)}}" frameborder="0" width="100%"
```

Comment:
if this is the only unique snippet you could merge with media-edit (maybe rename) and use ng-if there